Makefile

```diff
@@ -55,6 +57,19 @@ include Makefile.dapper
 # Make sure linting goals have up-to-date linting image
 $(LINTING_GOALS): package/.image.shipyard-linting
 
+scale-prereqs:
+	sudo sysctl -w fs.inotify.max_user_watches=1073741824
+	sudo sysctl -w fs.inotify.max_user_instances=524288
+	sudo sysctl -w kernel.pty.max=524288
+	sudo sysctl -w kernel.pid_max=524288
```

Comment:
This is smaller than the default in RHEL 8, Fedora 35/36, Debian 11 etc. Is that intentional?

More generally, could you describe why each setting is used? Some of them (`vm.swappiness` in particular) will have a lasting effect on the system’s behaviour, it would be good to warn the user about that.

Reply:
Thanks for the review, I updated it hope its clearer now